resources/views/partners/partners.blade.php

```diff
@@ -0,0 +1,31 @@
+@extends('layouts.layout')
+ 
+@section('content')
+<div class="row align-items-center">
+    <h1 class="col">Partners {{$year}}</h1>
+</div>
+<hr>
+<div class="container">
+    <div class="row row-cols-sm-4 g-4 justify-content-evenly">
+    @for($i = 0; $i < $groups->count(); $i++)
+        <!--Next row-->
+        <div class="col-sm">
+            <div class="card p-1 h-100">
+                <img src="{{$groups[$i]->imageurl}}" class="card-img-top" alt="Logo van {{$groups[$i]->name}}">
```

Comment:
Ik zou hier een img-fluid aan de class toevoegen

Reply:
Anders wordt de img helemaal in elkaar geperst. Maar dan moet je ook nog even zorgen dat de hoogte van de img en text goed blijft

Reply:
card-img-top doet hetzelfde als img-fluid, er verandert niks als die wordt toegevoegd/vervangen